### PR TITLE
Rename to @behance/beff, fix lint

### PR DESCRIPTION
**Pull request by hzoo**
I did a git clone and tried to `npm it` and got an eslint error about ecmaVersion so I updated. made another commit to rename

### CODE REVIEW

File: ux/scrollpoint.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ define([
              breakpoint * elementHeight($context) :
              parseInt(breakpoint, 10)
             )
-           );
+    );
```
Comment (eerikson) [inline ux/scrollpoint.js]: haha, did you just randomly have this open?

Comment (hzoo) [reply ux/scrollpoint.js]: no it errored in eslint lol

File: package.json [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
+    "@behance/hgn-loader": "^1.0.0",
+    "@behance/hogan.js": "^3.0.11",
+    "@behance/jquery-simulate": "1.0.1",
     "babel-polyfill": "^6.20.0",
     "eslint": "~3.4.0",
     "eslint-plugin-behance": "^1.0.0",
     "eslint-preset-behance": "^4.0.0",
-    "hgn-loader": "^0.1.1",
     "imports-loader": "^0.6.4",
-    "istanbul": "gotwarlost/istanbul#source-map",
```
Comment (hzoo) [inline package.json]: this didn't seem to do anything or I don't get it?

**Comment by hzoo** [comment]
Updated, removed the old package-lock, updated the subdeps

File: package.json (hunk 2 above)
Comment (Attamusc) [reply package.json]: Was this pinned for the loader below maybe?

Comment (hzoo) [reply package.json]: yeah supposedly and I made a package but I see coverage output without it

**Pull request merged**